### DISCUSSION TITLE
[Announcement][HW4] Re-grade for the Small test cases

### DISCUSSION THREAD

**Discussion by Junseon00** [prosyslab-classroom/cs348-information-security#443] [Announcements]
Hello Students, 

There was a mistake in grading the small test cases.(https://github.com/prosyslab-classroom/cs448-2021-spring/issues/141#issue-905330446)
As we noticed before, we'll re-grade the small test cases with the seed, that we first provided. `test/string_seed/input.txt`
Sorry for the confusion and we'll let you know when it's over. 

Also, as we noticed, we will **not** give you any json_parser which is used on evaluation.

Thank you.

**Comment by john-mai-2605** [comment]
Did the evaluation over. I ask because my score disappeared ...

**Comment by Junseon00** [comment]
I uploaded hw4 score now, so you can check your scores on KLMS
